_posts/2016-10-16-how-i-removed-two-databases-with-one-pull-request.md

```diff
@@ -0,0 +1,52 @@
+---
+title: How I removed two databases in one pull request
+authors: tge
+---
+
+My first project right after school was called KISS which means
```

Comment:
I'm not sure I understand how the _project_ itself can be called “KISS”. Isn't KISS is a practice? 🤔